### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by yuyu0927** [amyxlase/relpose-plus-plus#17]
Hi,

Hello, thank you for your wonderful work. I would like to ask, when the loss drops to what level does it mean that the model has basically converged?
My data contains 140 sequences. The current model is trained to 1640 iterations, the rotation loss is about 1.26, and the translation loss is about 0.14.

**Comment by jasonyzhang** [comment]
Those loss values seem reasonable. It may be more useful to look at the training time visualizations of the learned distributions.
